logic/account.js

```diff
@@ -585,8 +585,21 @@ Account.prototype.getAll = function (filter, fields, cb) {
 	delete filter.sort;
 
 	if (typeof filter.address === 'string') {
-		filter.address = {
-			$upper: ['address', filter.address]
+		filter['a.address'] = {
+			$upper: ['a.address', filter.address]
+		};
+		delete filter.address;
+	}
+
+	if (typeof filter.publicKey === 'string') {
+		filter.publicKey = {
+			$decode: ['publicKey', filter.publicKey, 'hex']
```

Comment:
Is there any documentation for these operators? I can't find any at https://github.com/LiskHQ/json-sql/tree/master/docs#condition-operators

Reply:
In Lisk we are using the fork of `json-sql` library which introduces new operators.